usaspending_api/references/models.py

```diff
@@ -377,13 +407,25 @@ class LegalEntity(DataSourceTrackedModel):
     small_business = models.CharField(max_length=1, blank=True, null=True)
     individual = models.CharField(max_length=1, blank=True, null=True)
 
+    def get_type_description(self):
+        description = [item for item in LEGAL_ENTITY_BUSINESS_TYPES if item[0] == self.business_types]
```

Comment:
That's a good idea. This code is similar in a lot of other places, so we'd want to refactor there at the same time. We should make another story for that, since the front-end needs this sooner than later :)